barter-data/src/exchange/bybit/message.rs

```diff
@@ -1,23 +1,25 @@
+use std::fmt::Debug;
+
 use crate::{
     Identifier,
-    event::MarketIter,
     exchange::bybit::{channel::BybitChannel, subscription::BybitResponse, trade::BybitTrade},
-    subscription::trade::PublicTrade,
 };
-use barter_instrument::exchange::ExchangeId;
 use barter_integration::subscription::SubscriptionId;
 use chrono::{DateTime, Utc};
 use serde::{
     Deserialize, Serialize,
     de::{Error, Unexpected},
 };
 
-/// [`Bybit`](super::Bybit) websocket message supports both [`BybitTrade`] and [`BybitResponse`].
+use super::book::BybitOrderBookMessage;
+
+/// [`Bybit`](super::Bybit) websocket message.
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum BybitMessage {
     Response(BybitResponse),
     Trade(BybitTrade),
+    Orderbook(BybitOrderBookMessage),
```

Comment:
Since `BybitMessage` is `serde(untagged)` it would be better to put the highest volume messages first (assuming this enum cannot be converted to be tagged). 

In practice barter-data uses one stream per SubKind, so this matters less. But that raises the question of creating a type like: 
```rust
pub enum BybitMessage<T> { 
    Response, 
    Data(T)
}
```